### PR TITLE
Merge master

### PR DESCRIPTION
**Pull request by MishaTy**
do not merge, building this would result in: `../hw/i2c/meson.build:20:7: ERROR: Unknown variable "arm_ss".` when running ../configure --enable-sdl --disable-cocoa --target-list=arm-softmmu --disable-capstone --disable-pie --disable-slirp --extra-cflags=-I/usr/local/opt/openssl@3/include --extra-ldflags='-L/usr/local/opt/openssl@3/lib -lcrypto' --disable-werror

### CODE REVIEW

**Comment by FrederickGeek8** [comment]
It's a little hard to parse all the commits, but do you know if this contains the `jackaudio` patches from upstream? I had to apply them in my fork to compile on my machine.

https://github.com/FrederickGeek8/qemu/commit/cc4a39b36b7e279ac8c441b1cb61296fb4a12246
https://gitlab.com/qemu-project/qemu/-/commit/ead789eb46a7df4eaab9e14e29e1d0d2a379988d

**Comment by MishaTy** [comment]
Closing this pr as I am moving github accounts
